web/src/main/java/org/cbioportal/genome_nexus/web/mixin/TranscriptConsequenceMixin.java

```diff
@@ -21,6 +21,10 @@
     @ApiModelProperty(value = "HGVSc", required = false)
     private String hgvsc;
 
+    @JsonProperty(value="hgvsg", required = true)
```

Comment:
Also add an annotation for this into `service/mixin/TranscriptConsequenceMixin.java` for consistency? I guess this still works because we always read from the cache. In case we have to read directly from the web service this field may be ignored. There is a `@JsonIgnoreProperties(ignoreUnknown = true)` in `service/mixin/TranscriptConsequenceMixin.java`.

Reply:
thanks - done